src/client/borchk.js

```diff
@@ -205,20 +210,23 @@ class LibrarySelector {
     });
   }
   select(element) {
-    var branchName = (element && element.name) || this.currentSearchValue;
-    var branchId = (element && element.branchId) || this.currentSearchValue;
-    var registrationUrl = (element && element.registrationUrl) || false;
+    if (element) {
+      var branchName = (element && element.name) || this.currentSearchValue;
```

Comment:
Du kan fjerne af element, da du har pakket det ind i en `if`, så bare `var branchName = element.name || this.currentSearchValue;`

Samme for det næste to linjer.